website_multi_theme/models/website.py

```diff
@@ -202,9 +202,11 @@ def _multi_theme_activate(self):
             views_to_remove = website.multi_theme_view_ids - custom_views
             # Removed views could be a copied parent for others
             # So, replace to original parent first
-            for view in self.env['ir.ui.view'].search([
-                    ('inherit_id', 'in', views_to_remove.ids)
-            ]):
+            for view in self.env['ir.ui.view']\
```

Comment:
Better for readability now to store in a variable the search result and then use it in the for.